draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -335,20 +332,84 @@ hop(AS(i), AS(j)) =  / Else, "Provider+" if the VAP-SPAS entry
         In the current document, routes with AS_SET are given Invalid evaluation in the AS_PATH verification procedures (<xref target="Upflow"/> and <xref target="Downflow"/>).
         See <xref target="mitigation"/> for how routes with Invalid AS_PATH are handled.
       </t>
+
+    <section title="Semantics of Valid, Invalid, and Unknown" anchor="semantics">
+      <t>
+        The ASPA-based path verification algorithms (<xref target="Upflow"/> and <xref target="Downflow"/>) evaluate an AS_PATH as Valid, Invalid, or Unknown.
+        'Invalid' outcome means that the AS_PATH contains a route leak (i.e., valley-free violation) based on the available ASPAs.
+        Even if the actual attack on the AS_PATH attribute was a forged-origin hijack or forged-path-segment hijack
```

Comment:
Agree, will remove the particular sentence.

Reply:
@cjeker 

> Why the path is not valley-free is out of scope since ASPA is unable to identify the cause.

This suggestion is followed in the revised Sec. 7 that I just sent by email for comments.